### PR TITLE
Add submodule instructions to readme

### PR DESCRIPTION
**Pull request by aaronjeline**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ To build the DRT framework:
 
 To run DRT:
 
+* First initialize the submodule containing the Cedar repository: `git submodule init && git submodule update`
```
[the remark this answers is not in the document]
Comment (aaronjeline) [reply README.md:28]: Ah didn't know that. Maybe we just call that out stronger? I tried to build DRT myself and ran into this issue.

[the remark this answers is not in the document]
Comment (khieta) [reply README.md:28]: We do not have instructions for that. Plenty of room for improvement in the README 😄 I feel that if you're going to call out the submodule initialization explicitly, then you should call out other steps hidden by `build.sh` too.

@aaronjeline do you want to take a general pass at improving the build instructions in this PR? If you're feeling motivated, there's also currently a minor-ish issue with the instructions/CI that I've been putting off: #41

[the remark this answers is not in the document]
Comment (aaronjeline) [reply README.md:28]: Sure thing

**Review by andrewmwells-amazon — APPROVED** [review]

**Comment by khieta** [comment]
Closing this issue since it's been inactive for a while. We can open a new PR in the future for documentation improvements.

**Pull request closed without merging**